### PR TITLE
CHANGELOG for 1.2.0

### PR DESCRIPTION
**Pull request by angelozerr**
CHANGELOG for 1.2.0

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Review by fbricon — CHANGES_REQUESTED** [review]
A bunch of issues seems to be missing from https://github.com/redhat-developer/vscode-quarkus/issues?utf8=%E2%9C%93&q=milestone%3A1.2.0+

**Comment by fbricon** [comment]
some issues are from the 1.1.1 release

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+## 1.2.0 (November 15, 2019)
+
+### Enhancements
+
+ * CodeAction to add required properties. See [quarkus-ls#111](https://github.com/redhat-developer/quarkus-ls/issues/111)
+ * CodeAction to fix unknown property by proposing similar name. See [quarkus-ls#80](https://github.com/redhat-developer/quarkus-ls/issues/80)
+ * Provide a better support for Quarkus property value. See [quarkus-ls#69](https://github.com/redhat-developer/quarkus-ls/issues/69)
+ * Use language instead of pattern for documentSelector. See [#145](https://github.com/redhat-developer/vscode-quarkus/pull/145)
+ * Use context global state to store defaults. See [#140](https://github.com/redhat-developer/vscode-quarkus/pull/140)
+ * Add quarkus extension to a gradle project. See [#112](https://github.com/redhat-developer/vscode-quarkus/issues/112)
```
Comment (fbricon) [inline CHANGELOG.md]: Gradle

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,31 @@
...
+ * Create new Gradle project. See [#108](https://github.com/redhat-developer/vscode-quarkus/issues/108)
```
Comment (fbricon) [inline CHANGELOG.md]: Create a new Gradle project

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md]: @fbricon updated

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md]: @fbricon updated

**Comment by angelozerr** [comment]
@fbricon anything else?

**Comment by fbricon** [comment]
let's hold on to that one until we have the proper release date

**Pull request merged**